jujugui/static/gui/src/app/components/entity-content/test-entity-content.js

```diff
@@ -114,6 +115,7 @@ describe('EntityContent', function() {
             </div>
           </div>
         </div>
+        {undefined}
```

Comment:
Yep! You got it. The JSX still creates an element, it is just undefined when we don't return the plans.